CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+## 15.19.0
+
+_released `2024-11-25`_
+
+For optimal performance, an optional argument has been added to [z.dehydrate](https://docs.zapier.com/platform/build-cli/overview#dehydration) to allow devs to control the caching duration on hydrators.
+
+Also, with the new `meta` field object added to the [FieldSchema](https://github.com/zapier/zapier-platform/blob/main/packages/schema/docs/build/schema.md#fieldschema) object, devs have more flexibility storing field-specific context which may be obtained when constructing dynamic needs, and then making that context available during actual action execution in the perform method.
```

Comment:
Adding "optional" working here as well? also changing the working to make it clear this is the expected behavior now, if needed

```suggestion
Also, with the new optional `meta` field object added to the [FieldSchema](https://github.com/zapier/zapier-platform/blob/main/packages/schema/docs/build/schema.md#fieldschema) object, devs now have a supported mechanism for storing field-specific context which may be obtained when constructing dynamic needs, and then making that context available during actual action execution in the perform method.
```

Reply:
Updated